lib/generators/rys/plugin/templates/%name%.gemspec.tt

```diff
@@ -1,6 +1,4 @@
-# encoding: utf-8
```

Comment:
proč?

Reply:
Utf8 je default od ruby 1.9, leda by si změnil file encoding a proč by to někdo dělal?

Reply:
Protože se to už stalo.

```ruby
s.author = ["Ondřej Mioravčík"] # => EncodingError
```

Reply:
na ruby2.5 + ? nebo kde ?

Reply:
to by asi chtělo příčinu proč to nastalo a na jakým ruby, ale tak to tam nech, ničemu to nevadí

Reply:
2.3 a řešení je přidat komentář